repos.json

```diff
@@ -4,20 +4,25 @@
     { "language": "cpp", "repo": "googleapis/google-cloud-cpp"},
     { "language": "cpp", "repo": "googleapis/google-cloud-cpp-common"},
     { "language": "cpp", "repo": "googleapis/google-cloud-cpp-spanner"},
+    { "language": "dotnet", "repo": "GoogleCloudPlatform/cpp-docs-samples"},
```

Comment:
This is actually a `C` and `C++` repository, I do not think there is any C# or any other .NET code in it.

Reply:
That's a great catch.  I think we've decided not to track this for now.